src/App.jsx

```diff
@@ -14,57 +15,53 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {good !== 'No goods selected' ? `${good} is selected` : good}
+        {good !== 'No goods selected' ? (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setGood('No goods selected')}
+          />
+        ) : null}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(value => {
+            return (
+              <tr
+                data-cy="Good"
+                key={value}
+                className={good === value ? 'has-background-success-light' : ''}
```

Comment:
consider moving `good === value` to a separate const with consistent name and use the const here